src/open_inwoner/scss/components/Actions/Actions.scss

```diff
@@ -9,130 +10,212 @@
+        max-width: 600px;
+      }
     }
 
     .button {
-      border: 1px solid;
-
-      &.actions__status-selector {
-        &--approval {
-          background-color: var(--color-danger-lightest);
-          border-color: var(--color-danger-lightest);
-          color: var(--color-danger);
-        }
-
-        &--open {
-          background-color: var(--color-info-lightest);
-          border-color: var(--color-info-lightest);
-          color: var(--color-info-darker);
-        }
-
-        &--closed {
-          background-color: var(--color-success-lighter);
-          border-color: var(--color-success-lighter);
-          color: var(--color-success);
-        }
+      padding: var(--spacing-medium);
+      span {
+        font-size: 24px;
+        max-width: 24px;
       }
     }
+  }
 
-    .dropdown__content .button {
-      border: none;
+  // Action data.
+  &__data {
+    display: grid;
+    gap: var(--spacing-giant) var(--spacing-mega);
+    grid-template-columns: 1fr;
+    grid-template-areas:
+      'for'
+      'status'
+      'created'
+      'execute';
+
+    @media (min-width: 992px) {
+      grid-template-columns: auto 1fr;
+      grid-template-areas:
+        'for created'
+        'status execute';
     }
   }
 
-  .actions__actions {
-    align-items: center;
-    position: absolute;
-    top: 0;
-    right: 0;
-    color: var(--color-gray-lighter);
-    font-family: var(--font-family-body);
-    font-size: var(--font-size-body);
-    display: flex;
-    justify-content: space-between;
+  // Remove first row if action is collapsed.
+  &[aria-expanded='false'] &__data {
+    grid-template-areas:
+      'status'
+      'execute';
+    @media (min-width: 992px) {
+      grid-template-areas: 'status execute';
+    }
+  }
+
+  // Action actions.
+  &__actions {
     display: flex;
     gap: var(--spacing-medium);
+    border-top: 1px var(--color-gray-light) solid;
+    padding-top: var(--spacing-large);
+    flex-direction: column;
 
-    [class*='icons'],
-    [class*='Icons'] {
-      color: var(--color-gray-lighter);
-      font-size: var(--font-size-body);
+    @media (min-width: 768px) {
+      flex-direction: row;
+      flex-wrap: wrap;
     }
   }
-  .button.actions__status-selector {
-    &--open {
-      color: var(--color-info-darker);
-      border-color: var(--color-info-darker);
+
+  // Action data row
+  &__label {
+    display: flex;
+    gap: var(--spacing-large);
+    align-items: center;
+
+    &--for {
+      grid-area: for;
     }
-    &--closed {
-      color: var(--color-success);
-      border-color: var(--color-success);
+    &--created {
+      grid-area: created;
     }
-    &--approval {
-      color: var(--color-danger);
-      border-color: var(--color-danger);
+    &--status {
+      grid-area: status;
+    }
+    &--execute {
+      grid-area: execute;
     }
-  }
 
-  &__footer {
-    .button {
+    &-value {
+      align-items: center;
+      color: var(--color-gray-dark);
```

Comment:
Note that `--color-gray-dark: rgba(75, 75, 75, 1);` is actually exactly the same color as `--color-body: var(--color-gray-dark);`
We may need to trim down the number of grays. But that would be a different issue.